pymc3/backends/base.py

```diff
@@ -40,6 +44,9 @@ def __init__(self, name, model=None, vars=None):
                            for var, value in var_values}
         self.chain = None
 
+    def values(self, point):
+        return (point[v] for v in self.varnames)
+
     ## Sampling methods
```

Comment:
I'd prefer to name this something else.  I think it looks too much like an attribute this way.